apps/arm/main.cpp

```diff
@@ -310,6 +315,7 @@ int main() {
   printf("ARM APPLICATION STARTED\r\n");
   printf("=======================\r\n");
 
+  initCAN();
```

Comment:
whats the point of having an initCAN function if it just calls the filter function? I think you should either not have it or you should move the other can initialization there as well (ex. tx rx thread spawning)

Reply:
Okay, sounds good